Created workflow connected to Github

nicolas-zampieri opened the pull request:
added .github/workflows folder with a laravel.yml, included in the .yml are all required dependency installations.\, requirements, and test to run. #87

nicolas-zampieri committed:
Created workflow

added .github/workflows folder with a laravel.yml

vinnythepoo2 reviewed (approved):
Good job with the testing setup

JonathanFrantzeskos reviewed (approved):
good start for testing setup

nicolas-zampieri committed:
Updated laravel.yml to include proper directory for running test

nicolas-zampieri committed:
Commented out tests

These are tests that come with laravel framework, commented out for CI pipeline confirmation.

nicolas-zampieri committed:
Update test case

Uncommented for the purpose of finding the class
